### ISSUE TITLE
Apps or scripts are not served on sub-paths

### ISSUE THREAD

**Issue by aranvir** [h2oai/wave#1895]
#### Wave SDK Version, OS
wave server 0.25.0
h2o-wave 0.25.1
python 3.10.7
windows 10

#### Actual behavior

When I run any app or script (e.g., the [system monitor example](https://wave.h2o.ai/docs/tutorial-monitor)) on a route like `/stats/monitor` instead of `/monitor`, nothing is shown. The waved server does receive data periodically:
```
2023/03/16 08:33:41 * /stats/monitor {"d":[{"k":"cpu_stats data usage","v":0.0},{"k":"cpu_stats plot_data -1","v":[16,0.0]},{"k":"mem_stats data usage","v":93.0},{"k":"mem_stats plot_data -1","v":[16,93.0]}]}
2023/03/16 08:33:43 * /stats/monitor {"d":[{"k":"cpu_stats data usage","v":10.5},{"k":"cpu_stats plot_data -1","v":[17,10.5]},{"k":"mem_stats data usage","v":93.0},{"k":"mem_stats plot_data -1","v":[17,93.0]}]}
2023/03/16 08:33:45 * /stats/monitor {"d":[{"k":"cpu_stats data usage","v":9.6},{"k":"cpu_stats plot_data -1","v":[18,9.6]},{"k":"mem_stats data usage","v":93.0},{"k":"mem_stats plot_data -1","v":[18,93.0]}]}
```
But when I visit the page at `http://localhost:10101/stats/monitor` it's just a white background as if no widgets were loaded. This is in contrast to visiting any "single" route like `/stats` where the "wave live placeholder" is shown instead. If I set the script path to `/monitor` it works fine. For apps, it's the same behavior: Setting the route to `/` or `/app` is fine but `/app/first` will just show an empty window.

Interestingly, when I set the script path to `/#/stats/monitor` or `/#/monitor` it works! The same does not work for the app though.

#### Expected behavior

Based on https://wave.h2o.ai/docs/routing, I expected that can choose any "composite" route as an endpoint (from example: `/foo and /foo/bar are two distinct paths.`.

My idea was to have an app that hosts multiple pages with different forms but also self-refreshing dashboards that are served on `/scripts/` and are pulled into the app with `ui.frame()`. This is really just for clean separation of app pages and script views, so it would be a nice to have.

If the actual behavior IS the expected behavior, I think it would be nice to mention that explicitly in the documentation. Or even better: Add my expectation as a feature 😄 😉 

#### Steps To Reproduce

1. Run system monitor script at `/monitor`. Visit `http://localhost:10101/monitor` to confirm it's working
2. Run system monitor script at `/stats/monitor`. Visit `http://localhost:10101/stats/monitor`. You will see an empty page. Check the server log to confirm that data is being sent.
3. Run system monitor script at `/#/stats/monitor`. Visit `http://localhost:10101/#/stats/monitor` to confirm it's working

**Comment by mturoci** [comment]
Hi @aranvir this should be already fixed in 0.25.1. Can you please try it out?

**Comment by mturoci** [comment]
>wave server 0.25.0
h2o-wave 0.25.1

I just noticed. This is the problem. Wave server and python driver need to always match versions.
